frontend/src/components/WaitingRoom/VideoContainer/VideoContainer.tsx

```diff
@@ -60,7 +60,7 @@ const VideoContainer = ({ username }: VideoContainerProps): ReactElement => {
 
   return (
     <div
-      className="relative flex w-[584px] max-w-full flex-col items-center justify-center bg-black sm:h-[328px] md:rounded-xl"
+      className="relative flex aspect-video w-[584px] max-w-full flex-col items-center justify-center bg-black sm:h-[328px] md:rounded-xl"
```

Comment:
TIL I learned about `aspect-video`: https://tailwindcss.com/docs/aspect-ratio

and they really had to rick-roll us